src/test/java/scenarios/TransactionTest.java

```diff
@@ -661,29 +691,32 @@ public void a2verifyThatTransactionListAppears() {
         Set<String> setAfterFilter = new HashSet<>(transactionIDAfterFilterApply);
         System.out.println(setAfterFilter);
 
+
         // Check  setAfterFilter contains all elements from setBeforeFilter
         Assertions.assertTrue(setAfterFilter.containsAll(setBeforeFilter));
 
         session.getTransactionsPage().getFilterIcon().click();
         // Click on Download button
 
-        WebdriverWaits.sleep(5000);
         String fileStatus = ActionEngine.isFileDownloaded("Transactions.xlsx");
```

Comment:
Improvement Suggestion: Validate file content after download to ensure the file is correct.